### PR TITLE
Update emotion examples with new API for SSR and React endpoint

### PR DESCRIPTION
**Pull request by hamlim**
This PR fixes this issue [#80](https://github.com/tkh44/emotion/issues/80) opened up at https://github.com/tkh44/emotion.

The `emotion/server` export changed names and the react `styled` imports changed as well. I also manually bumped the version of emotion in the `package.json` file.

It looks like something in my config changed the indentation significantly. Not sure what could have caused that but if anyone has tips on how to resolve let me know!

### CODE REVIEW

**Review by tkh44 — CHANGES_REQUESTED** [review]
It looks like the file formatting is off. You should be able to format the code with `npm run lint`

File: examples/with-emotion/pages/index.js [hunk 1]
```diff
@@ -69,15 +69,11 @@ export default () => {
...
-        Cool Styles
-      </Basic>
+      <Basic>Cool Styles</Basic>
       <Combined>
         With <code>:hover</code>.
       </Combined>
-      <Animated animation={bounce}>
-        Let's bounce.
-      </Animated>
+      <Animated animation={bounce}>Let's bounce.</Animated>
```
Comment (hamlim) [inline examples/with-emotion/pages/index.js]: Looks like prettier made this change to one line, I can revert this back if you'd like. cc @tkh44

**Comment by hamlim** [comment]
Fixed the tabs vs spaces issue, this should be good to be reviewed again @tkh44.

**Review by tkh44 — APPROVED** [review]

**Review by timneutkens — APPROVED** [review]
Nice! 💯

**Pull request merged**